src/Features/Core/Portable/ConvertToInterpolatedString/AbstractConvertPlaceholderToInterpolatedStringRefactoringProvider.cs

```diff
@@ -27,6 +28,17 @@ internal abstract class AbstractConvertPlaceholderToInterpolatedStringRefactorin
         where TLiteralExpressionSyntax : SyntaxNode
         where TArgumentListExpressionSyntax : SyntaxNode
     {
+
+        // Methods that are not string.Format but still should qualify to be replaced.
+        // Ex: Console.WriteLine("{0}", a) => Console.WriteLine($"{a}");
+        private static readonly (string? typeName, string[] methods)[] TypeMethodPairs = new (string?, string[])[]
```

Comment:
I think it would be much more meaningful if this was called `CompositFormattingMethods` or something. `TypeMethodPairs` is very generic, and if the docs use "composite formatting" for this pattern, it would make sense to mirror that in code.